src/__experimental__/components/textbox/textbox.component.js

```diff
@@ -196,6 +198,11 @@ Textbox.propTypes = {
   adaptiveLabelBreakpoint: PropTypes.number,
   /** Flag to configure component as required */
   required: PropTypes.bool,
+  /**
```

Comment:
I don't think this should be hidden, it's used by some implementation teams and would be good to document it. The type is also different here than in Search?